airbyte-cdk/bulk/toolkits/extract-cdc/src/main/kotlin/io/airbyte/cdk/read/cdc/CdcPartitionReader.kt

```diff
@@ -92,7 +93,8 @@ class CdcPartitionReader<T : Comparable<T>>(
         val summary: Map<String, Any?> =
             mapOf(
                     "debezium-version" to debeziumVersion,
-                    "records" to numRecords.get(),
+                    "emitted records" to numEmittedRecords.get(),
```

Comment:
curious: Where are those magic strings come from? Like debezium constants?

Reply:
they don't come from anywhere, they're just there for log messages, nothing else